arches_lingo/src/arches_lingo/components/scheme/SchemeLabel/components/SchemeLabelEditor.vue

```diff
@@ -80,17 +91,28 @@ async function save(e: FormSubmitEvent) {
 
         openEditor!(props.componentName, updatedTileId);
     } catch (error) {
-        console.error(error);
+        toast.add({
+            severity: ERROR,
+            life: DEFAULT_ERROR_TOAST_LIFE,
+            summary: $gettext("Failed to save data."),
+            detail: error instanceof Error ? error.message : undefined,
+        });
     } finally {
         refreshReportSection!(props.componentName);
+        isSaving.value = false;
     }
 }
 </script>
 
 <template>
     <h3>{{ props.sectionTitle }}</h3>
```

Comment:
nit: spinner doesn't hide section title

Reply:
The fixe is to handle <h3> separately. I am not sure if it would be better <h3> and <Form> should be in one <div v-show=...>.

Reply:
yeah I'd say `<template v-show-...>`